tools/RAiDER/processWM.py

```diff
@@ -35,6 +35,9 @@ def getWMFilename(weather_model_name, time, outLoc):
         )
     )
 
+    if weather_model_name is 'GMAO' or 'MERRA2':
+        f = f[:-2]+'h5'
+
```

Comment:
@leiyangleon i fixed the "osr" issue we have and confirmed it went fine before i merged this one. I believe this change is impacting one of the unit test to fail for ERA5. Can you have a closer look?